packages/EasyErrorHandler/src/DataTransferObjects/ErrorCodeDto.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\DataTransferObjects;
+
+final class ErrorCodeDto
+{
+    /**
+     * @param array<int, string> $splittedName
+     */
+    public function __construct(
+        private string $originalName,
+        private int $errorCode,
+        private array $splittedName,
+        private ?string $groupSeparator = null
+    ) {
+    }
+
+    /**
+     * @return array<int, string>
+     */
+    public function getSplittedName(): array
```

Comment:
Please change `Splitted` to `Split`, it's a more correct form.

Reply:
Done